R/tm_missing_data.R

```diff
@@ -141,15 +141,16 @@ ui_page_missing_data <- function(id, datasets, pre_output = NULL, post_output =
   )
 }
 
-srv_page_missing_data <- function(id, datasets, reporter, plot_height, plot_width, ggplot2_args) {
+srv_page_missing_data <- function(id, data, reporter, filter_panel_api, plot_height, plot_width, ggplot2_args) {
```

Comment:
Deselecting all variables from the combinations tab crashes the app:
![Screenshot from 2022-09-02 13-51-56](https://user-images.githubusercontent.com/86970066/188138796-40badc9b-027b-4eb6-a8b0-ef52d8077812.png)

Reply:
fixed